templates/server/src/main/resources/archetype-resources/core/src/test/java/__packageInPathFormat__/general/common/base/test/ApplicationComponentTest.java

```diff
@@ -10,7 +10,7 @@
 /**
  * Abstract base class for {@link ComponentTest}s of this application.
  */
-@SpringBootTest(classes = { SpringBootApp.class }, webEnvironment = WebEnvironment.NONE)
+@SpringBootTest(classes = { SpringBootApp.class })
```

Comment:
not very explicitly but convention over configuration (default is `WebEnvironment.MOCK`).

Reply:
You are definitively right. Sry, could not see this comment in my app :/